tests/e2e/utils/__fixtures__/price-benchmark-suggestions.json

```diff
@@ -1,15 +1,19 @@
 [
 	{
 		"product": {
-			"id": 259251,
+			"id": 33,
```

Comment:
@joemcgill I update this ID to match a product with the same ID in my local setup.